mozilla-mobile/android-components/components/feature/addons/src/main/res/values/strings.xml

```diff
@@ -170,8 +170,18 @@
     <string name="mozac_feature_addons_successfully_installed">Successfully installed %1$s</string>
     <!-- Text shown after failed to install an add-on. %1$s is the add-on name. -->
     <string name="mozac_feature_addons_failed_to_install">Failed to install %1$s</string>
+    <!-- Text shown after failing to install an add-on for which we don't have its name. -->
+    <string name="mozac_feature_addons_failed_to_install_generic">Failed to install this add-on.</string>
+    <!-- Text shown when attempting to install an add-on and a network error happened. -->
+    <string name="mozac_feature_addons_failed_to_install_network_error">This add-on could not be downloaded because of a connection failure.</string>
+    <!-- Text shown when attempting to install an add-on and the downloaded file is corrupted. -->
+    <string name="mozac_feature_addons_failed_to_install_corrupt_error">This add-on could not be installed because it appears to be corrupt.</string>
+    <!-- Text shown when attempting to install an add-on and an error occurred because the extension was not signed. -->
+    <string name="mozac_feature_addons_failed_to_install_not_signed_error">This add-on could not be installed because it has not been verified.</string>
+    <!-- Text shown when attempting to install an add-on and an error occurred because the extension was incompatible. %1$s is the add-on name, %2$s is the app name and %3$s is the app version. -->
+    <string name="mozac_feature_addons_failed_to_install_incompatible_error">%1$s could not be installed because it is not compatible with %2$s %3$s.</string>
     <!-- Text shown when attempting to install a blocklisted add-on. %1$s is the add-on name. -->
-    <string name="mozac_feature_addons_blocklisted">%1$s could not be installed because it has a high risk of causing stability or security problems</string>
+    <string name="mozac_feature_addons_blocklisted">%1$s could not be installed because it has a high risk of causing stability or security problems.</string>
```

Comment:
This is not OK.
https://github.com/mozilla-mobile/firefox-android/pull/3332/files#r1324011199

To work around it, I'm going to manually fix the file here, so that I can merge and expose these strings for localization

Reply:
Sorry, about that I put a PR to address the issue https://github.com/mozilla-mobile/firefox-android/pull/3615